scripts/spot-grid.py

```diff
@@ -53,12 +61,30 @@ def new_image(self, data):
         at ratio 1:1)
         data (numpy.ndarray): an 2D array containing the image (can be 3D if in RGB)
         """
-        self.app.spots, trans, scale, rot, shear = FindGridSpots(data, self.gridsize)
-        self.app.translation = trans[0], data.shape[0] - trans[1]
-        self.app.scale = scale
-        self.app.rotation = -rot
-        self.app.shear = shear
-
+        tform_ji, _ = estimate_grid_orientation_from_img(
+            data,
+            (8, 8),
```

Comment:
you can use `self.gridsize` instead of (8, 8), to allow compatibility with images that have grids with different sizes.

Reply:
thanks